sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSessionReceiverAsyncClient.java

```diff
@@ -138,7 +138,7 @@ public Mono<ServiceBusReceiverAsyncClient> acceptNextSession() {
             .map(sessionId -> {
                 final ReceiverOptions newReceiverOptions = new ReceiverOptions(receiverOptions.getReceiveMode(),
                     receiverOptions.getPrefetchCount(), receiverOptions.getMaxLockRenewDuration(),
-                    receiverOptions.isEnableAutoComplete(), sessionId, null);
+                    receiverOptions.isEnableAutoComplete(), sessionId, null, receiverOptions.getSessionIdleTimeout());
```

Comment:
iiuc, idle timeout is applicable only when rolling is in action (in other words, when max-concurrent-session is not null / > 1). In the case of `acceptNextSession`  there is no rolling but one session, if so, should we use the other ReceiverOptions Ctr that does not have idle-timeout?

Reply:
Similar comment for `acceptSession(String sessionId)`